.github/workflows/build.yml

```diff
@@ -28,10 +28,10 @@ jobs:
           - 21
           - 25
         module:
-          - api
-          - client
-          - testing
-          - testcontainers
+          - docling-api
```

Comment:
Since the idea is to possibly have a `docling-client` module in the future using Docling directly (no Docling Serve like in `docling-serve-client`), I wonder if we should similarly have a `docling-serve-api` module (for all the code currently in docling-api) and a `docling-api` module that would only include the Docling core APIs (e.g. `Document`). In alternative, single module, but two sub packages (`core`, `serve`)?

Reply:
So your thinking is that the request/response objects are specific to the serve API and not the core API? 

If so I see where you are going with this. I think though at this point we don't have a clue about what an implementation without docling-serve will look like, so I'd hate to make things more complicated. 

That being said, I'm not opposed to it either. There will most likely be some iterations on this, so whatever we do at this point surely won't be frozen in time. Until we get to a 1.0 we can change whatever we'd like.